core/tests/protractor/accessibility.js

```diff
@@ -54,3 +54,40 @@ describe('Cache Slugs', function() {
     general.checkConsoleErrorsExist(expectedErrors);
   });
 });
+
+describe('Meta Tags', function() {
+  var GET_STARTED_URL = '/get_started';
+  var GET_STARTED_META = {
```

Comment:
Prefer declaring as two simple constants:

EXPECTED_META_NAME

EXPECTED_META_DESCRIPTION

(The word "EXPECTED" is important.)

Reply:
Done